docs/pages/setup/reference/cli.mdx

```diff
@@ -743,6 +745,46 @@ add_headers:
       "Authorization": "Bearer tokentokentoken"
 ```
 
+#### Aliases
+
+You can define custom `tsh` command aliases using the `aliases` section. It also allows for changing of the default flag values for existing commands.
+
+New command `tsh l`:
+
+```yaml
+aliases:
+    "l": "tsh login --auth=okta --username=alice"
+```
+
+Make `tsh status` use JSON as a default format:
+
+```yaml
+aliases:
+    "status": "tsh status --format=json"
+```
+
+Define custom command using with `bash`. The `$0` and `$1` variables will be substituted by command arguments.
+
+```yaml
+aliases:
+    "connect": "bash -c 'tsh login $0 && tsh ssh $1'"
+```
+
+The alias can use an arbitrary number of arguments. If a argument variable `$N` is referenced, `tsh` will check that at least `N+1` arguments were given to the alias invocation. All arguments that were given but not referenced in the alias definition will be appended at the end.
+
+Given configuration:
+
+```yaml
+aliases:
+    "example": "bash -c 'echo first=$0 $0-$1 $3'"
+```
+
+`tsh example 0 1 unused-2 3 unused-4` will expand to `bash -c 'echo first=0 0-1 3 unused-2 unused-4'`.
+
+To debug aliases, set the `TELEPORT_DEBUG=1` environment variable.
+
+To disable an alias temporarily, set the `TSH_ALIAS` environment variable. `tsh` understands the value of that flag to be a comma-separated list of aliases it should ignore. For example, `TSH_ALIAS=connect,status` will make it behave as if those two aliases were not set. This environment variable is also used to prevent infinite alias loops from occurring.
```

Comment:
`TSH_ALIAS` sounds like you're enabling something, not disabling something. I would recommend changing the implementation to `TSH_DISABLE_ALIASES` or something..

Reply:
This is an implementation detail though; I will remove it from the docs. Do you find `TSH_UNALIAS` a better name? Wink to http://www.linfo.org/unalias.html.

Reply:
See: https://github.com/gravitational/teleport/pull/14931/commits/fb89a2c9872818e21bc5d3d03fdc0aa61baf22be